getting_started.md

```diff
@@ -4,19 +4,50 @@ permalink: /gettingstarted/
 title: Getting Started
 ---
 
-There are two ways to use thiss: Either point your SP to an existing service instance or run your own instance. Most SPs will want to use an existing service instance but some may need or want to run their own instance of thiss - for instance to have full control over the IdPs listed in the discovery service. 
+The Coalition for Seamless Access provides the thiss service to enable several different “flavors” of implementation:
 
-# Using an existing service
+* **Limited** - lets you use the Seamless Access discovery service for users to find and sign into their preferred Identity Provider, but doesn’t integrate this service into your site.
+* **Standard** - lets you use the Seamless Access service to display the button on your site, and use the Seamless Access discovery and persistence services as integrated components on your site.
+* **Advanced** - provides you with the Seamless Access persistence service while giving you greater control over the appearance of the service on your site, and what Identity Providers (IdPs) you include in your discovery service.
 
-In order to connect to an existing service instance typically means follow instructions similar to [the instructions for how to connect to the demo service](/use/).
+# Using the Seamless Access Service or thiss Software
 
-# Running your own instance
+To implement the Seamless Access **Limited** or **Standard** 'flavors', you will be using the Seamless Access service which is a hosted instance of the thiss software. This service provides the core benefits of Seamless Access with minimal work. Most Service Providers (SPs) will prefer this option.
 
-Runing your own instance of thiss-js involves two things:
+To implement the **Advanced** features, you will need to use the thiss software to run your own instance of the service.
+
+# Getting Started - "Limited" Flavor
+
+For the the "Limited" flavor, you will implement:
+
+* Discovery Service Integration - _Integrate the Seamless Access service to use it as your Discovery Service._
+
+See the [Integration Guide](/integration/) to learn how to configure your Service Provider (SP) software stack.
+
+# Getting Started - "Standard" Flavor
+
+For the "Standard" flavor, you will implement:
+
+* Discovery Service Integration - _Integrate the Seamless Access service to use it as your Discovery Service._
+* Display of Seamless Access Login Button - _Use the Seamless Access service to display the login button component on your SP login page._
+* Integration of Login Button with your SAML SP - _Integrate the Seamless Access service to use it as your Discovery Service._
+
+See the [Integration Guide](/integration/) for details on how to implement.
+
+# Getting Started - "Advanced" Flavor
```

Comment:
I wasn't sure if this was still accurate. I tried doing some update in the implementation guide version, but want to discuss before this section gets finalized.